x-pack/plugins/security_solution/public/entity_analytics/components/risk_engine_privileges_callout/risk_engine_privileges_callout.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useMemo } from 'react';
+import hash from 'object-hash';
+import type { CallOutMessage } from '../../../common/components/callouts';
+import { CallOutSwitcher } from '../../../common/components/callouts';
+import { MissingPrivilegesCallOutBody, MISSING_PRIVILEGES_CALLOUT_TITLE } from './translations';
+import { useMissingPrivileges } from './use_missing_risk_engine_privileges';
+
+export const RiskEnginePrivilegesCallOut = () => {
+  const missingPrivileges = useMissingPrivileges();
+
+  const MissingPrivilegesMessage: CallOutMessage | null = useMemo(() => {
```

Comment:
This variable name seems to be a component, can we rename it to not start with a capital letter?

Reply:
Fixed in [0435e57](https://github.com/elastic/kibana/pull/171250/commits/0435e57531ff4bfeb33ac902e5a12b25ac1295c2)